src/AccessibilityInsights/Scripts/UIAccess.cmd

```diff
@@ -0,0 +1,42 @@
+@echo off
+ if /i '%1' == 'Enable' goto Confirm
+ if /i '%1' == 'Disable' goto Confirm
+ 
+ :Usage
+ echo.
+ echo UIAccess.cmd
+ echo.
+ echo This script will set the UIAccess status of Accessibilty Insights for Windows.
+ echo UIAccess support is disabled by default. This script must be run from an
+ echo administrative command prompt.
+ echo.
+ echo Usage: UIAccess.cmd ^<action^>
+ echo.
+ echo Where ^<action^> is one of:
+ echo   ENABLE  (to enable UIAccess support)
+ echo   DISABLE (to disable UIAccess support)
+ echo.
+ goto :eof
+ 
+ :Confirm
+ echo.
+ echo CAUTION! You should only change the UIAccess state if you understand the
+ echo          potential security implications!
+ echo.
+ choice /m "Are you sure you want to %1 UIAccess support?"
+ if ErrorLevel 1 goto UpdateManifest
+ 
+ :UserCanceled
+ echo.
+ echo Operation was canceled.
+ echo.
+ goto :eof
+ 
+ :UpdateManifest
+ rem DO NOT use copy here, as we also need to update the file's last write time
+ type UIAccess_%1d.manifest > AccessibilityInsights.exe.manifest
+ echo.
+ echo Update complete. Changes will take effect when Accessibilty Insights for
```

Comment:
I don't know if we care enough about this script to fix this case, but if the type command fails we still get this success message. I hit it because I attempted to run the script without CDing into the folder:
![image](https://user-images.githubusercontent.com/4615491/61547915-386f0580-aa01-11e9-8555-688f487eefae.png)

Reply:
Same with non admin command prompt window. It does however also say that `Access is denied`
![image](https://user-images.githubusercontent.com/45673569/61551334-ba632c80-aa09-11e9-827f-d1c8e587bc5d.png)

Reply:
@flyingUnderTheRadar, I've updated the script to cover both the case where it's run from a different folder, and the case where it's run without administrative access. I also fixed the script to not update if the user says "No" at the confirm dialog.

---
In reply to: [305437587](https://github.com/microsoft/accessibility-insights-windows/pull/444#discussion_r305437587) [](ancestors = 305437587)

Reply:
I've fixed this

---
In reply to: [305415488](https://github.com/microsoft/accessibility-insights-windows/pull/444#discussion_r305415488) [](ancestors = 305415488)

Reply:
LGTM ! I tried running it from a different folder and with non admin. Works as it is supposed to